packages/fuels/src/cli/commands/deploy/deployContract.ts

```diff
@@ -0,0 +1,27 @@
+import { ContractFactory } from '@fuel-ts/contract';
+import type { DeployContractOptions } from '@fuel-ts/contract';
+import type { WalletUnlocked } from '@fuel-ts/wallet';
+import { readFileSync } from 'fs';
+
+import { debug } from '../../utils/logger';
+
+export async function deployContract(
+  wallet: WalletUnlocked,
+  binaryPath: string,
+  abiPath: string,
+  deployConfig: DeployContractOptions
+) {
+  debug(`Deploying contract for ABI: ${abiPath}`);
+
+  const bytecode = await readFileSync(binaryPath);
```

Comment:
```suggestion
  const bytecode = readFileSync(binaryPath);
```

`await` has no effect here.